static/documentation/forking.md

```diff
@@ -0,0 +1,38 @@
+# Working with a forked repo
+
+## Forking the repo
+
+In order to contribute to this project you will need to create a fork of the repo on your own account. 
+To fork the repo just open the repository in a browser and click on the "fork" button on the top right of the page.
+
+If you are part of any organisations it will ask you which organisation you want to create the fork in.
```

Comment:
Missing comma:
**If you are part of any organisations, it will ask you which organisation you want to create the fork in.**

Reply:
Updated as suggested.